src/main/java/com/hcl/appscan/jenkins/plugin/builders/AppScanBuildStep.java

```diff
@@ -294,8 +295,23 @@ private void perform(Run<?,?> build, Launcher launcher, TaskListener listener) t
     	m_authProvider = new JenkinsAuthenticationProvider(m_credentials, build.getParent().getParent());
     	final IProgress progress = new ScanProgress(listener);
     	final boolean suspend = m_wait;
-    	final IScan scan = ScanFactory.createScan(getScanProperties(build, listener), progress, m_authProvider);
-        
+        Map<String, String> properties = getScanProperties(build,listener);
+    	final IScan scan = ScanFactory.createScan(properties, progress, m_authProvider);
+        JenkinsAuthenticationProvider checkAppScan360 = (JenkinsAuthenticationProvider) m_authProvider;
```

Comment:
Instead of having to call `checkAppScan360.isAppScan360()` numerous times throughout the code, this could be simplified to:
`boolean isAppScan360 = ((JenkinsAuthenticationProvider)m_authProvider).isAppScan360();`
This also makes the code easier to read.

Reply:
Okay, done.